python/lib/evaluations/dmod/evaluations/crosswalk/disk.py

```diff
@@ -30,7 +30,10 @@ def get_format(cls) -> str:
 
     def retrieve(self, *args, **kwargs) -> pandas.DataFrame:
         crosswalked_data = reader.select_values(self._document, self.field)
-        crosswalked_data.dropna(inplace=True)
+
+        if not (crosswalked_data is None or crosswalked_data.empty):
+            crosswalked_data.dropna(inplace=True)
+
```

Comment:
It's because the `select_values` is a general purpose json scanner, not specifically for the crosswalk stuff specifically (anymore). Another json scanner was being used by the crosswalk data _and_ the data retriever, but it got too complicated to fit the crosswalk's needs, so _it_ ended up using its own, then the data retriever one got moved over as well since the new one was better. The `select_values` function (and the file that contains it) needs to get promoted closer to the root of the project, but it lacked priority. Either way, due to its general purpose, it returning an empty set is "reasonable", hence the check. I'm going to push an update that moves the file in a bit.